src/Navbar/index.js

```diff
@@ -9,7 +9,11 @@ export default () => (
         {!window.email ?
           <li className="navigation-item"><Link to='/login' className="navigation-link">Login</Link></li>
           :
-          <li className="navigation-item"><Link to='/logout' className="navigation-link">Logout</Link></li>
+          <li className="navigation-item">
+          <Link to='/' className="navigation-link">Hello {window.email}&nbsp;&nbsp;</Link>
```

Comment:
Indent when you are defining a child (`<Link>`) of an element (`<li>`)

Reply:
Also I would like that you make 2 commits as this is another issue. Atomic commits help going back in the project history

Reply:
Sorry my bad, you already made two. When making the commits, also give a brief description of the issue as in 
>    Add username in navbar (Fixes #2)